io.github.tofudd.lanchat.terminal.json

```diff
@@ -0,0 +1,67 @@
+{
+  "app-id": "io.github.tofudd.lanchat.terminal",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "20.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.dotnet5"
+  ],
+  "finish-args": [
+    "--share=network",
+    "--env=DOTNET_ROOT=/app/dotnet/lib",
+    "--filesystem=home"
```

Comment:
Ideally your app should respect XDG specification to store configuration files in XDG_CONFIG_HOME and data in XDG_DATA_HOME.